lib/iex/lib/iex/helpers.ex

```diff
@@ -827,6 +828,154 @@ defmodule IEx.Helpers do
 
   defp pad_key(key), do: String.pad_trailing(key, 21, " ")
 
+  @process_info_keys_and_labels [
+    {:initial_call, "Initial call"},
+    {:dictionary, "Dictionary"},
+    {:registered_name, "Registered name"},
+    {:current_function, "Current function"},
+    {:status, "Status"},
+    {:message_queue_len, "Message queue length"},
+    {:trap_exit, "Trap exit"},
+    {:priority, "Priority"},
+    {:group_leader, "Group leader"},
+    {:reductions, "Reductions"},
+    {:links, "Links"},
+    {:monitors, "Monitors"},
+    {:memory, "Memory"},
+    {:total_heap_size, "Total heap size"},
+    {:heap_size, "Heap size"},
+    {:stack_size, "Stack size"},
+    {:current_stacktrace, "Current stacktrace"}
+  ]
+  @process_info_keys Enum.map(@process_info_keys_and_labels, fn {key, _} -> key end)
+  @process_info_label_mapping Map.new(@process_info_keys_and_labels)
+
+  @doc """
+  Prints information about the given process.
+  """
+  @doc since: "1.19.0"
+  def process_info(pid) when is_pid(pid) do
+    with true <- Process.alive?(pid),
+         info when is_list(info) <- Process.info(pid, @process_info_keys) do
```

Comment:
This will allow us to trace any process or name or via tuple, locally or on a remote node.

```suggestion
  def process_info(pid) do
    with pid when is_pid(pid) <- GenServer.whereis(pid),
         info when is_list(info) <- :erpc.call(node(pid), :erlang, :process_info, [pid, @process_info_keys]) do
```